lib/xandra/connection.ex

```diff
@@ -265,8 +263,7 @@ defmodule Xandra.Connection do
         {:error, {:connection_crashed, reason}}
     after
       timeout ->
-        :telemetry.execute([:xandra, :client_timeout], %{}, telemetry_metadata)
```

Comment:
We don't need this, clients can measure this themselves for now if they want. I want to first run this code in your codebase and see how things go before making this a public Telemetry event.

Reply:
@harunzengin it probably will be in the future, but I want to avoid committing to an API for the telemetry event until we're sure.